### DISCUSSION TITLE
Cloud egress charge for DANDIsets?

### DISCUSSION THREAD

**Discussion by bingxinghuo** [dandi/helpdesk#123] [Q&A]
Hi there! May I know the cost associated with moving DANDIsets from where they are (presumably AWS) to other cloud platforms (GCP, Azure)? Does standard egress fee apply? If we are setting up features to do this routinely, can there be more cost-effective arrangements? Thanks!

**Comment by CodyCBakerPhD** [comment]
Download charges from the S3 bucket is handled by the same open data AWS program that makes the bucket storage free to users, so no cost to you.

Bandwidth usage by your other cloud resources depends on the associated charges and rates by those services

What I can say and recommend is that if you use AWS resources, such as EC2, there is no additional charge for bandwidth on either end **as long as your compute instances are in the same region as the DANDI Archive (us-east-2)** (for instance, how the DANDI Hub platform is setup)

I'd advise consulting [https://docs.aws.amazon.com/cur/latest/userguide/cur-data-transfers-charges.html](https://docs.aws.amazon.com/cur/latest/userguide/cur-data-transfers-charges.html) for general advice
